src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -321,6 +338,35 @@ const AccessibilityRequestDetailPage = () => {
           }}
         </Formik>
       </div>
+      <div
+        role="region"
+        aria-label="existing notes"
+        className="margin-top-6 margin-x-1"
+      >
+        <div
```

Comment:
I'm not sure why this `div` is needed. The region in the `div` above is already labeled with "existing notes".

Reply:
That's how I interpreted the Figma design, but sounds like that might not be the right interpretation

Reply:
Sounds like it was meant to be an h3, and the X is the number of notes.